documentation/modules/deploying/con-deploy-operator-best-practices.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// deploying/assembly_deploy-tasks-prep.adoc
+
+[id='con-deploy-operator-best-practices-{context}']
+= Operator deployment best practices
+
+[role="_abstract"]
+Potential issues can arise from installing more than one Strimzi operator in the same Kubernetes cluster, especially when using different versions.
+Each Strimzi operator manages a set of resources in a Kubernetes cluster. 
+When you install multiple Strimzi operators, they may attempt to manage the same resources concurrently. 
+This can lead to conflicts and unpredictable behavior within your cluster.
+Conflicts can still occur even if you deploy Strimzi operators in different namespaces within the same Kubernetes cluster. 
+While namespaces provide some level of resource isolation, certain resources managed by the Strimzi operator, such as Custom Resource Definitions (CRDs) and roles, are cluster-scoped. 
```

Comment:
I think the ibm style mentions refraining from using "While" when not related to time

```suggestion
Although namespaces provide some degree of resource isolation, certain resources managed by the Strimzi operator, such as Custom Resource Definitions (CRDs) and roles, possess a cluster-wide scope.
```

Reply:
makes sense